packages/cactus-plugin-ledger-connector-quorum/package.json

```diff
@@ -64,14 +67,17 @@
     "sanitize-html": "2.7.0",
     "typescript-optional": "2.0.1",
     "web3": "1.5.2",
-    "web3-eth-contract": "1.5.2"
+    "web3-eth-contract": "1.5.2",
+    "minimist": "1.2.8"
   },
   "devDependencies": {
     "@hyperledger/cactus-plugin-keychain-memory": "1.1.3",
     "@hyperledger/cactus-test-tooling": "1.1.3",
     "@types/express": "4.17.13",
     "@types/sanitize-html": "2.6.2",
-    "web3-eth": "1.5.2"
+    "@types/minimist": "1.2.2",
+    "web3-eth": "1.5.2",
+    "chalk": "4.1.2"
```

Comment:
@outSH Please make sure that the dependencies are ordered alphabetically! (silly nit-pick I know, apologies)

Reply:
@petermetz Sure thing :D I've used https://www.npmjs.com/package/sort-package-json on entire project and there are more files out of order, I can create an issue to fix them all at once and maybe add this check to CI && run it on configure

Reply:
@outSH Dejavu haha. There's this blast from the past ( https://github.com/hyperledger/cacti/issues/1341 ) that we did to do exactly what you just described but I see now that it's been broken this whole time because it does the package.json sorting but then it doesn't actually report an error if the package.json was unsorted, it just sorts it but that means that when it runs on the CI the changes it made to sort things are never actually committed.

We could make it part of the configure script, but with all the checks that it runs it would add an extra 20 seconds to the (already too slow) build so I'll suggest that we just fix it so that the CI fails if there are unsorted package.json files AND advise people to run the check manually before they open their PRs (so that they don't wait hours for the CI to pass but then it fails because of the sorting check).

See `tools/custom-checks/check-package-json-sort.ts` for the implementation (that is currently broken because it's supposed to error out if there were unsorted package.json files)

Reply:
> Done btw.

Cheers!